build.sh

```diff
@@ -99,6 +99,7 @@ build_omi
 copy_version_file
 
 echo "Building OMI Python Script provider"
+export CPPFLAGS="-fpermissive"
```

Comment:
This should be removed!

Reply:
ho, just see this now, but this is in your commit...
you can remove it next time, thanks.